src/libraries/System.Private.CoreLib/src/System/Globalization/CultureData.Unix.cs

```diff
@@ -78,14 +78,52 @@ private bool InitCultureDataCore()
             return true;
         }
 
+        private void InitUserOverride(bool useUserOverride)
+        {
+            // Unix doesn't support user overrides
+            _bUseOverrides = false;
```

Comment:
Aside (not this PR) it would be nice at some point to remove all the ugly Hungarian prefixes in CultureData

Reply:
Yeah, I agree, that'd be a nice cleanup.

Reply:
Hungarian notations is actually to match Windows exposed API's fields. I am not pushing again the clean up but wanted to tell why we had it in the first place.